hal/armv7a/imx6ull/_init.S

```diff
@@ -31,6 +31,9 @@
 #define ADDR_TTL2_K   (ADDR_TTL1 + 4 * SIZE_PAGE)
 #define ADDR_TTL2_EXC (ADDR_TTL2_K + SIZE_PAGE)
 
+#define ADDR_PROGS_BEGIN (_end + 4 * 1024 * 1024 - VADDR_KERNEL + ADDR_DDR)
+#define ADDR_SYSPAGE     (ADDR_DDR + 0x20)
```

Comment:
We don't really need this define - we rather need syspage offset, but it's easy enough to just type in `syspage_data - init_vectors`